rules/windows/builtin/security/win_scheduled_task_deletion.yml

```diff
@@ -19,7 +20,9 @@ logsource:
 detection:
     selection:
         EventID: 4699
-    condition: selection
+    falsepositive1:
+        TaskName: '\Microsoft\Windows\RemovalTools\MRT_ERROR_HB' # triggered by ParentCommandLine=C:\WINDOWS\system32\MRT.exe /EHB /HeartbeatFailure ErrorStack,Previous=ErrorStack,Previous=ErrorStack,Previous=ErrorStack,Previous=ErrorStack,Previous=SubmitHeartbeatReportData,Hr=0x80072f8f,Hr=0x80072f8f,Hr=0x80072f8f,Hr=0x80072f8f,Hr=0x80072f8f /HeartbeatError 0x80072f8f
```

Comment:
Are you sure it's a 1:1 match on not a substring? (`TaskName|contains: ...`)

Reply:
Yes, I am sure. I have 15 entries from 15 different machines this morning.

Reply:
```
2022-05-16 08:23:47 REDACTED Security-Auditing: 4699: A scheduled task was deleted. | SubjectUserSid=S-1-5-18 | SubjectUserName=REDACTED$ | SubjectDomainName=REDACTED | SubjectLogonId=0x3e7 | TaskName=\Microsoft\Windows\RemovalTools\MRT_ERROR_HB | TaskContent= | ClientProcessStartKey=5066549580802483 | ClientProcessId=10952 | ParentProcessId=1464 | RpcCallClientLocality=0 | FQDN=REDACTED | pid=716 | level=0 | sys_version=1 | category=Other Object Access Events | op=Info | Security=true | key=0x8020000000000000 | id=34812216 | app="C:\Windows\System32\lsass.exe" | tid=6932 | channel="Security" | type=audit success(0) |
```